pinot-segment-local/src/test/java/org/apache/pinot/segment/local/segment/index/loader/ForwardIndexHandlerTest.java

```diff
@@ -387,7 +464,7 @@ public void testComputeOperation()
     segmentLocalFSDirectory.close();
   }
 
-  @Test
+  @Test(priority = 1)
```

Comment:
Why is priority needed ?

Reply:
Because we are changing raw columns to dict enabled columns in some newer tests. We want these enable dictionary tests to run after all the test for changing compression types (which work on raw columns) have run.